lib/hypernova/blank_renderer.rb

```diff
@@ -35,4 +35,14 @@ def name
   def id
     @id ||= SecureRandom.uuid
   end
+
+  def html_attributes
+    # handles content_tag()-like options
+    attributes = ''
+    options = job[:html_options]
+    if options && options[:class]
+      attributes << %{ class="#{options[:class]}"}
```

Comment:
what if `options[:class]` includes a double quote? please include a test case that covers this.